packages/pn-commons/src/components/NotificationDetail/NotificationRelatedDowntimes.tsx

```diff
@@ -75,6 +75,12 @@ const NotificationRelatedDowntimes = (props: Props) => {
 
   const [shouldFetchEvents, setShouldFetchEvents] = useState<boolean>(false);
 
+  const isCancelled = !!props.notificationStatusHistory.find(
```

Comment:
this is in commons. so you must pass the variable disableDownloads as done for the others